### PR TITLE
[ Wait for #1428 ] [ GRU ] Add drop out mask 

### PR DESCRIPTION
**Pull request by jijoongmoon**
This commit includes:
  . Implementation of drop out mask

Resolves:

**Self evaluation:**
1. Build test:	 [X]Passed [ ]Failed [ ]Skipped
2. Run test:	 [X]Passed [ ]Failed [ ]Skipped

Signed-off-by: jijoong.moon <jijoong.moon@samsung.com>

### CODE REVIEW

File: nntrainer/layers/gru.cpp [hunk 1]
```diff
@@ -67,6 +75,11 @@ void GRULayer::finalize(InitLayerContext &context) {
   output_dim = input_dim;
   output_dim.width(unit);
 
+  if (dropout_rate > epsilon) {
+    wt_idx[GRUParams::dropout_mask] = context.requestTensor(
+      output_dim, "GRU:dropout_mask", true, ITERATION_LIFESPAN);
```
Comment (kparichay) [inline nntrainer/layers/gru.cpp:80]: `true` -> `false`
https://github.com/nnstreamer/nntrainer/pull/1427#discussion_r676238140

Comment (jijoongmoon) [reply nntrainer/layers/gru.cpp:80]: I will fix.

**Review by kparichay — COMMENTED** [review]
LGTM

**Review by zhoonit — APPROVED** [review]
LGTM

**Review by kparichay — APPROVED** [review]
LGTM

**Pull request merged**